.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
 - bash scripts/gogetcookie.sh
 - go get github.com/kardianos/govendor
 script:
-- make vendor-status test vet
+- make vet vendor-status test
```

Comment:
(I've fixed the merge conflict here and reverted this back to how it was in `96617b609bf7abd0979b9c099b91cc79feac4461`)